taxonomy-editor-frontend/src/pages/home/index.jsx

```diff
@@ -0,0 +1,63 @@
+import Button from '@mui/material/Button';
+import {Link as MuiLink} from '@mui/material';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import Container from '@mui/material/Container';
+import { Link } from "react-router-dom";
+
+function Copyright() {
+  return (
+    <Typography variant="body2" color="text.secondary" align="center" sx={{ mt: 8, mb: 4 }}>
+      {'Copyright © '}

```

Comment:
nit: this text doesn't needs to be inside {''}

Reply:
I'm using that convention to add a space after the copyright logo